services/offline/src/lib/cacheable-section-state.js

```diff
@@ -37,14 +37,25 @@ export function createCacheableSectionStore() {
     return createStore(initialState)
 }
 
+/**
+ * Helper hook that returns a value that will persist between renders but makes
+ * sure to only set its initial state once.
+ */
+function useConstRef(factory) {
+    const ref = React.useRef(null)
+    if (ref.current === null) ref.current = factory()
```

Comment:
Our code style prefers not to use single-line if statements - curly brackets should always be preferred, so that it is very explicit

Reply:
Ah - didn't know! I admit I've violated that guideline pretty often... 🙈